app/views/templates/_template.html.slim

```diff
@@ -0,0 +1,11 @@
+tr
+  td class="question"
```

Comment:
Similar table was in question so I re-used class "question". But now it is excessive, so I removed it